scripts/cms-unsupported/readme.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+if (check_file_exists('README.md')) {
+    $contents = read_file('README.md');
+    $rx = "#\n\[!\[Silverstripe supported module\].+?\n#si";
```

Comment:
```suggestion
    $rx = "#\n?\[!\[Silverstripe supported module\].+?\n#si";
```
What if there's no new line before the badge? We still want to remove it.

Reply:
Updated